en_US/configuration/configuration.md

```diff
@@ -149,6 +149,30 @@ The value of HOCON will be overridden hierarchically, the rules are as follows:
 - In the same file, the value defined in the later section will override any previous key value.
 - A higher-level value will replace that of a lower-level.
 
+The EMQX configuration is prioritized in the following order: `environment variables > emqx.conf > API(cluster.hocon)`.
+```
+                    Start
+                      |
+            Check Environment Variables
+                 /          \
+             Found         Not Found
+              |                  |
+           Use Env        Check emqx.conf
+                           /             \
+                        Found           Not Found
+                          |                  |
+                    Use emqx.conf    Check cluster.hocon
+                                          /          \
+                            Use cluster.hocon      Use Default Values
+
+```
+
+Settings in environment variables that begin with 'EMQX_' have the highest priority and will override any settings in the `etc/emqx.conf` file.
+
+Changes made through the EMQX dashboard UI, management HTTP API, or CLI will be written into the `data/configs/cluster.hocon` file at runtime and will take effect immediately.
+
+However, if the same configuration items are set differently in the `etc/emqx.conf` file, the runtime updates will be overridden by the settings in `etc/emqx.conf` during a restart.
```

Comment:
```suggestion
However, if the same configuration items are set differently in the `etc/emqx.conf` file, the runtime updates will be overridden by the settings in `etc/emqx.conf` after the node restarts.

To avoid confusion, it is highly recommend NOT to have the same config keys in both `cluster.conf` and `emqx.conf`.
```